include/zephyr/net/http/table.h

```diff
@@ -0,0 +1,107 @@
+/** @file
+ * @brief HTTP request methods
```

Comment:
@Emna-Rekik - please clean up the doxygen to reflect the "table" tag / name (or whatever the HPACK library is).